Core/src/org/sleuthkit/autopsy/geolocation/GeolocationUtilities.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package org.sleuthkit.autopsy.geolocation;
+
+import java.util.logging.Level;
+import org.sleuthkit.autopsy.coreutils.Logger;
+import org.sleuthkit.datamodel.BlackboardArtifact;
+import org.sleuthkit.datamodel.BlackboardAttribute;
+import org.sleuthkit.datamodel.TskCoreException;
+
+/**
+ * Helper function for managing the getting artifact information.  These functions
+ * were grabbed from KMLReport.
+ * 
+ */
+class GeolocationUtilities {
+    private static final Logger logger = Logger.getLogger(GeolocationUtilities.class.getName());
+    /**
+     * Get a Double from an artifact if it exists, return null otherwise.
+     *
+     * @param artifact The artifact to query
+     * @param type     The attribute type we're looking for
+     *
+     * @return The Double if it exists, or null if not
+     */
+    static Double getDouble(BlackboardArtifact artifact, BlackboardAttribute.ATTRIBUTE_TYPE type) {
+        Double returnValue = null;
+        try {
+            BlackboardAttribute bba = artifact.getAttribute(new BlackboardAttribute.Type(type));
+            if (bba != null) {
+                Double value = bba.getValueDouble();
```

Comment:
Any specific reason for this two step assignment?  
Couldn't returnValue  be directly assigned to?

(applies to other utilities methods in the class)